browser/resources/settings/brave_clear_browsing_data_dialog/brave_clear_browsing_data_on_exit_page.ts

```diff
@@ -3,85 +3,115 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at https://mozilla.org/MPL/2.0/.
 
-// @ts-nocheck
-
-import {Polymer} from 'chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js';
-import {loadTimeData} from '../i18n_setup.js';
-import '../settings_shared.css.js';
-import '../settings_vars.css.js';
-import '../controls/settings_checkbox.js';
+import {PrefsMixin} from '/shared/settings/prefs/prefs_mixin.js'
```

Comment:
wow great work @emerick! It's not always straightforward doing these conversions :laughing:

Reply:
Thank you! A couple were tricky but it got a little easier over time. I'm not sad it's over though, lol.